app/src/main/res/drawable/thumb_pluralsight.xml

```diff
@@ -0,0 +1,27 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:width="200dp"
+    android:height="200dp"
+    android:viewportWidth="200"
+    android:viewportHeight="200">
+  <path
+      android:pathData="M99.34,100.29m-100,0a100,100 0,1 1,200 0a100,100 0,1 1,-200 0">
```

Comment:
The circle is a bit off and peaks out of the viewport. Maybe try `M100,100M0,100a100,100 0,1 1,200 0,100 100,0 1,1 -200,0` for the **pathData** instead.